h2/src/main/org/h2/expression/ValueExpression.java

```diff
@@ -33,6 +34,16 @@
      */
     private static final Object DEFAULT = new ValueExpression(ValueNull.INSTANCE);
 
+    /**
+     * The expression represents ValueBoolean.TRUE.
+     */
+    private static final Object TRUE = new ValueExpression(ValueBoolean.TRUE);
```

Comment:
why make these of type Object and then cast them in the code?

Reply:
Because `TestClearReferences` complains about them otherwise.